Nimble-Snapshots.podspec

```diff
@@ -12,9 +12,22 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "8.0"
   s.pod_target_xcconfig = { 'ENABLE_BITCODE' => 'NO' }
   s.source       = { :git => "https://github.com/ashfurrow/Nimble-Snapshots.git", :tag => s.version }
-  s.source_files  = "HaveValidSnapshot.swift", "PrettySyntax.swift", "NimbleSnapshotsConfiguration.swift"
+  s.default_subspec = "Core"
   s.frameworks  = "Foundation", "XCTest"
-  s.dependency "FBSnapshotTestCase", "~> 2.0"
-  s.dependency "Nimble"
-  s.dependency "Quick"
+
+  s.subspec "Core" do |ss|
+    ss.source_files  = "HaveValidSnapshot.swift", "PrettySyntax.swift", "NimbleSnapshotsConfiguration.swift"
+    ss.dependency "FBSnapshotTestCase", "~> 2.0"
+    ss.dependency "Nimble"
+    ss.dependency "Quick"
+  end
+
+  s.subspec "DynamicType" do |ss|
+    ss.source_files  = "DynamicType/*.{swift,m,h}"
+    # ss.private_header_files = "DynamicType/MockedApplication.h"
```

Comment:
If I uncomment this, the Swift file can't access that class. Is there a way to make this visible to our library but hidden for outside consumers?